package/CHANGELOG

```diff
@@ -20,6 +20,8 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
   * 0.21.0
 
 Fixes
+  * PCA(align=True) now correctly aligns the trajectory and computes the
```

Comment:
add your GitHub handle to the author list for "0.21.0"

Reply:
👍